### PR TITLE
Home page

### PR DESCRIPTION
**Pull request by dis-may**
## Context

We need a home page

## What Changed?

* Added custom localhost theme with colours and fontFamily (`tailwind.config.ts` file)
* Changed components to include these themed values (touched header, footer, cards) instead of being hardcoded certain colours
* Added background (the green colour is not a div overlay because it goes over text, so its just part of the background image)
* Made a new `Heading` component which looks like a navbar but has no buttons and central text
* put the home page together as closely as I could to the Figma 

## How To Review

See if the website looks right, if its responsive, buggy, etc. Also if i made any questionable coding choices

## Testing

It should run, none of the links go anywhere but it runs and is somewhat responsive to smaller sizes.

## Notes

### CODE REVIEW

File: src/app/home/page.tsx
Comment (codecreator127) [inline src/app/home/page.tsx:1]: what's the mainText and subText for?

File: src/app/layout.tsx
Comment (codecreator127) [inline src/app/layout.tsx:1]: is there supposed to be the footer component in both layout.tsx and the page.tsx?

**Review by codecreator127 — CHANGES_REQUESTED** [review]
I think you have some duplicate code, but I took a look at it and it looks pretty good apart from that

File: src/app/home/page.tsx
Comment (dis-may) [reply src/app/home/page.tsx:1]: oops left it in accidentally from trying something, ill remove it

File: src/app/layout.tsx
Comment (dis-may) [reply src/app/layout.tsx:1]: it was removed from the `layout.tsx` page, and i put it in `page.tsx` instead

**Review by codecreator127 — APPROVED** [review]
looks good. go ahead and merge

**Pull request merged**